src/Xendit.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Xendit;
+
+/**
+ * Class Xendit
+ *
+ * @package Xendit
+ */
+class Xendit
+{
+    public static $apiKey;
+
+    public static $apiBase = 'https://api.xendit.co';
```

Comment:
It's good to have a getter & setter for this as well so we can use this library against staging servers etc